graphql_compiler/compiler/emit_sql.py

```diff
@@ -80,60 +81,54 @@ def _traverse_and_validate_blocks(ir: IrAndMetadata) -> Iterator[BasicBlock]:
         yield block
 
 
-def _find_columns_used_outside_folds(
+def _find_columns_used(
```

Comment:
nit/personal preference: I'd change this function name to `_find_used_columns`. Sound more readable to me for some reason.